extra_docs/jfs/README.md

```diff
@@ -0,0 +1,209 @@
+# Jexia Fileset Module (JFS)
+
+JFS is used for the uploading, fetching and manipulating files as any other user data. Basically, fileset has the same 
+functionality as dataset, but provides a way to maintain large data objects (files).  
+Before uploading any file you need to manually create a fileset with web management application 
+[app.jexia.com](https://app.jexia.com).
+You can also set AWS bucket and credentials so that your files will be automatically uploaded to the Amazon storage.
+
+### [Initialize](#init) 
+To use fileset features, just add fileOperations module to the client initialization. Notice, that for Node.js
+you have to import fileOperations from *jexia-sdk-js/node*, but if you running your application in browser you need to
+use *jexia-sdk-js/browser* path. It's because file format is different under these two platforms.
+```typescript
+import { jexiaClient, fileOperations } from 'jexia-sdk-js/node'; // 'jexia-sdk-js/browser' for browser applications
+
+const jfs = fileOperations();
+
+jexiaClient().init({
+  projectID: 'your-project-id',
+  key: 'your-project-key',
+  secret: 'your-project-secret'
+}, jfs);
+```
+
+### [Configure module](#configure)
+You can pass a config object to the `fileOperation()` function:
+```typescript
+const jfs = fileOperations({
+  /* Use RTC module to subscribe for the fileset events and wait until file
+     will be successfully uploaded. Needs RTC module to be activated
+     default: false */
+  subscribeForTheFileUploading: boolean
+  /* Timeout for uploading files
+     default: 120000 */
+  uploadTimeout: number
+});
+```
+
+### [Files format](#format)
+Under Node.js `jfs` accepts [read stream](#https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_class_fs_readstream), 
```

Comment:
we should [add `engines` to `package.json`](https://docs.npmjs.com/files/package.json#engines) to match this node version

Reply:
well, as long as our SDK can be used also with browser, I don't think we need to add node as an engine. NPM will require node then, do we want this?

Reply:
we could add browsers too, but I suggest we discuss that afterwards :+1: